NebulaWorld/MonoBehaviours/Local/ChatManager.cs

```diff
@@ -0,0 +1,130 @@
+﻿using NebulaModel.Logger;
+using NebulaModel.Packets.Players;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace NebulaWorld.MonoBehaviours.Local
+{
+    public class ChatManager : MonoBehaviour
+    {
+        public InputField chatBox;
+        public int maxMessages = 25;
+        public GameObject chatPanel, textObject, notifier, chatWindow;
+        [SerializeField] private List<Message> messages = new List<Message>();
+        public Color playerMessage, info;
+        private int _attemptsToGetLocationCountDown = 25;
+        private bool _sentLocation;
+
+        void Update()
+        {
+            if (chatBox.text != "")
+            {
+                if (Input.GetKeyDown(KeyCode.Return))
+                {
+                    if (Multiplayer.IsActive)
+                    {
+                        Multiplayer.Session.Network.SendPacket(new NewChatMessagePacket(ChatMessageType.PlayerMessage,
+                            chatBox.text, DateTime.Now, GetUserName()));
+                    }
+                    else
+                    {
+                        Log.Debug($"Chat message is only sent locally");
```

Comment:
Does it make sense to enable the chat in a single player game?

Reply:
Good point, probably not, I was mostly using that for testing. Will remove